src/render-html/css.js

```diff
@@ -149,6 +149,10 @@ th, td {
   white-space: nowrap;
   background: #ccc;
 }
+.mentioned {
+  padding-top: 4px;
+  background: hsl(8, 94%, 94%);
```

Comment:
Few problems:
* `hsl` is not compatible with older browsers.
* `rgba` is better than `hsl` because it allows you to use alpha to blend in the different light vs dark theme
* we likely don't even want the same styling